### PR TITLE
Grafana local deployment documentation

### PR DESCRIPTION
**Pull request by Mihaylov93**
# Description

* Added documentation on how to deploy Edge Analytics dashboard to a local instance of Grafana.
* Added repository config files to deploy such documentation with Github Pages.

Preview of Github Pages: https://mihaylov93.github.io/edge-analytics-grafana-setup/
After merging to main will be available on: https://system73.github.io/edge-analytics-grafana-setup/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
-# edge-analytics-grafana-setup
-Self-hosted Grafana integration with Edge Analytics
+# Local Grafana Deployment
```
Comment (layoaster) [inline README.md]:
```suggestion
# Self-hosted Grafana integration with Edge Analytics
```

File: docs/files/ea-basic-dashboard.json [hunk 2]
```diff
@@ -0,0 +1,2642 @@
...
+      "5m",
+      "15m",
+      "30m",
+      "1h",
+      "2h",
+      "1d"
+    ]
+  },
+  "timezone": "",
+  "title": "Edge Intelligence - Basic",
```
Comment (layoaster) [inline docs/files/ea-basic-dashboard.json]:
```suggestion
  "title": "Custom Edge Analytics ",
```

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    wrapping around the image. Instead, `block` format will force the text to be above or below the
+    image, but never to the sides. */
+    display:block; 
+    float:none; 
+    margin-left:auto;
+    margin-right:auto;
+}
+</style>
+<!-- markdownlint-enable -->
+# Local Grafana Deployment
```
Comment (layoaster) [inline docs/index.md]:
```suggestion
# Self-hosted Grafana integration with Edge Analytics
```

Comment (layoaster) [inline docs/index.md]:
```suggestion
# Self-hosted Grafana integration with Edge Analytics
```

File: .markdownlint.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# markdownlint-cli configuration file (YAML)
+# markdownlint rules: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md
```
Comment (layoaster) [inline .markdownlint.yaml:2]: How do you pass the linter to enforce these rules ?

Comment (Mihaylov93) [reply .markdownlint.yaml:2]: I ran it manually, but I just commited a pre-commit hook to enforce it before commits.

https://github.com/System73/edge-analytics-grafana-setup/pull/1/commits/d9c9f507ed4395d6b10324a7573850f5d41bcd2e

**Review by layoaster — APPROVED** [review]

**Pull request merged**